### DISCUSSION TITLE
Voir le shelter d'une autre guilde

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#576] [Nouvelles idées]
Voir le refuge d'une guilde dont on ne fait pas partie.

**Comment by niqore** [comment]
Je suis contre, le shelter c'est privé

**Reply by Pietagorh** [reply]
Bah y'a rien à cacher dans un shelter et en plus c'est pratique d'avoir accès au shelter des autres

**Reply by niqore** [reply]
C'est surtout d'un point de vue RP, les gens extérieurs à une guilde ne sont pas censés avoir accès au shelter

**Reply by DeadAngelV6** [reply]
Gros RP sur draftbot hein xD

**Reply by BastLast** [reply]
Justifier l'absence de logique par l'absence de logique, c'est non.

**Reply by DeadAngelV6** [reply]
Oulah je justifie rien dutout, j'dis juste qu'utiliser le terme "RP" pour draftbot c'est pas trop le truc.

C'est surtout car c'est personnel comme une banque (ce que c'est en soit)
Contre aussi, un shelter doit rester privé.
